### ISSUE TITLE
Autocomplete for path arguments does not add " " or escape whitespaces when needed

### ISSUE THREAD

**Issue by foxyseta** [ulyssa/iamb#175]
If a path argument features whitespaces, " " should be automatically added by autocompletion. Either that or the whitespaces should be escaped with \\.

**Comment by VAWVAW** [comment]
fixed by ulyssa/modalkit#186

**Comment by foxyseta** [comment]
Indeed! I upgraded to `iamb 0.0.12-alpha.2 (fc1c164)`, which uses `modalkit 0.0.26`, and this fixed the issue. Thank you!
